packages/core/src/components/Card/Content.tsx

```diff
@@ -215,21 +243,6 @@ export default withStyles(({ color, pattern, ui, unit }) => ({
     paddingLeft: unit * 1.5,
   },
 
-  image: {
```

Comment:
top image uses these `image` styles. please bring them back! that's why the top image happo diffs were off

Reply:
Huh, so weird that this didn't affect my local styleguide at all.

Reply:
removing it squished the image instead of covering the space. would be hard to notice unless you have that image memorized! ha

Reply:
Locally it actually looked the same as the after in happo, that's the weird thing.

Reply:
interesting. to debug the happo stuff, i cloned your repo and pulled this branch. locally i'm seeing the broken top image (which is the same as the "after" in the happo):

<img width="1279" alt="Screen Shot 2019-09-27 at 1 25 56 PM" src="https://user-images.githubusercontent.com/306275/65800099-789de300-e12a-11e9-9584-99848c5653d3.png">

since it's missing the `objectFit: 'cover'`, it doesn't look as intended:
http://airbnb.io/lunar/?path=/story/core-card--a-card-with-a-top-featured-image